kubernetes/deployment-staging.tmpl

```diff
@@ -115,6 +120,13 @@ spec:
               mountPath: "/static-assets"
             - name: caesar-nginx-config
               mountPath: "/etc/nginx-sites"
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 80
+              initialDelaySeconds: 5
+              periodSeconds: 3
+              successThreshold: 1
```

Comment:
Generally, do we want to specify options when we're just using the default values? Personally I don't think we really need to, but it's fine if other people find it clearer that way.

Reply:
I left it in for clarity here, happy to remove if folks find it confusing.